bin/circleci/build-frontend.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -ex
-if [[ " $TESTPILOT_ENABLE_PONTOON_BRANCHES " =~ " $CIRCLE_BRANCH " ]]; then
-    NODE_ENV=production ENABLE_PONTOON=1 npm run static
+if [[ $CIRCLE_BRANCH == 'master' ]]; then
+    NODE_ENV=production ENABLE_PONTOON=1 ENABLE_DEV_CONTENT=1 npm run static
```

Comment:
I replied over there - tl;dr: local & remote dev environments have enough differences to separate them, and IMO I've found turning these things on/off individually to be handy.